### DISCUSSION TITLE
为什么搭建的节点手机移动开热点可以用，但是用wifi就不行呀

### DISCUSSION THREAD

**Discussion by lzmopadoor** [v2fly/v2ray-core#2547] [Q&A]
如果你遇到的问题不是 V2Ray 的 bug，比如你不清楚要如何配置，请使用[Discussion](https://github.com/v2fly/v2ray-core/discussions)进行讨论。

此 Issue 会被立即关闭。

If you are not sure if your question is truely a bug in V2Ray, please discuss it [here](https://github.com/v2fly/v2ray-core/discussions) first.

This issue will be closed immediately.

**Comment by wangqingkeyi** [comment]
我也遇到了这个问题，楼主解决了吗

**Comment by suucss** [comment]
这个问题不是 V2Ray 的 bug，请移步 Discussion 进行讨论。请检查一下网络连通性（手机和 WIFI 分别测试），比如使用 `ping` 等工具，看看是不是手机热点能连上节点，而 WIFI 连不上。如需进一步讨论请移步 Discussion 并提供更多信息。

**Comment by lzmopadoor** [comment]
> 我也遇到了这个问题，楼主解决了吗

没有，不过听别人说重启路由可能有效，还有可能是域名的原因，不过我都试过了，都不行，你可以试试运气
